shipment.py

```diff
@@ -302,12 +302,25 @@ def send_nacex(cls, api, shipments):
                 errors.append(message)
                 continue
 
+            redone_label = None
             if values[0] == 'ERROR':
-                message = gettext(
-                    'carrier_send_shipments_nacex.msg_nacex_not_send_error',
-                    name=shipment.rec_name, error=resp.text)
-                errors.append(message)
-                continue
+                if values[2] == '5016':
+                    resp = nacex_call(api, 'editExpedicion', data)
+                    if len(values) == 1 or resp.status_code != 200:
```

Comment:
Line 307, the values list has min 2 elements. Why len(values) == 1 when values has min 2 elments?